core/evaluation.py

```
@@ -0,0 +1,396 @@
+
+    EXACT = "exact"
+    CONTAINS = "contains"
+    TOOLS = "tools"
+    TOOL_CALLS = "tool_calls"  # 别名
+    SEMANTIC = "semantic"
+
+
+# 别名 - 向后兼容
+TaskDifficulty = Difficulty
+VerificationType = VerifyType
+EvaluationTask = Task
```

Ruff (0.15.10)

[error] Undefined name `Task`

(F821)